### PR TITLE
Add autotools snap to the testing flow of wix-ui-core

### PR DESCRIPTION
**Pull request by Zombiefruit**

### CODE REVIEW

File: packages/wix-ui-core/src/components/Button/Button.meta.ts [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
-  });
\ No newline at end of file
+
+buttonMetadata.exportedFrom({
+  path: 'src/components/Button/Button',
+  exportName: 'Button',
+  baseStylePath: 'src/components/Button/Button.st.css'
+});
+
+buttonMetadata.addStyle(style, {name: 'style', path: 'src/components/Button/ButtonStyle.st.css'});
```
Comment (shlomitc) [inline packages/wix-ui-core/src/components/Button/Button.meta.ts]: please move the `ButtonStyle.st.css` file to the `stories/Button` folder to maintain same structure across the different files (like `circularProgressBar`)

File: packages/wix-ui-core/src/components/Checkbox/Checkbox.meta.ts [hunk 2]
```diff
@@ -1,7 +1,17 @@
...
 
 const checkboxMetadata = Registry.getComponentMetadata(Checkbox);
+
+checkboxMetadata.exportedFrom({
+  path: 'src/components/Checkbox/Checkbox',
+  exportName: 'Checkbox',
+  baseStylePath: 'src/components/Checkbox/Checkbox.st.css'
+});
+
+checkboxMetadata.addStyle(style, {name: 'style', path: 'src/components/Checkbox/CheckboxStyle.st.css'});
```
Comment (shlomitc) [inline packages/wix-ui-core/src/components/Checkbox/Checkbox.meta.ts]: same as `Button`

File: packages/wix-ui-core/src/components/Button/Button.meta.ts (hunk 1 above)
Comment (Zombiefruit) [reply packages/wix-ui-core/src/components/Button/Button.meta.ts]: Sure

File: packages/wix-ui-core/scripts/run-snap.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+const {exec} = require('child_process');
+
+if (!process.env.APPVEYOR) {
```
Comment (shlomitc) [inline packages/wix-ui-core/scripts/run-snap.js]: please change this to
```js
if (process.env.EYES_API_KEY) {
  execSync('npm run snap', { stdio: 'inherit' });
}
```

**Comment by Zombiefruit** [comment]
@hepiyellow That's the hope. We need to merge another PR and then this one is next

**Comment by JoMarton** [comment]
@shlomitc @Zombiefruit 
Do you have a specific single screen size? Or you want it configurable? If configurable then I suggest we hold this feature to another PR. It requires discussing the API to do that and I don't want to hold up this PR.

I would also prefer to not hold up this PR over console colors but if it affects functionality your choice

**Comment by shlomitc** [comment]
@JoMarton - We currently have a fixed size, but it can be configurable.
I agree with your suggestion - making the screen size as we have today and make it configurable in another PR

**Pull request merged**